src/workerd/util/mimetype.c++

```diff
@@ -6,24 +6,64 @@
 #include <kj/debug.h>
 #include <kj/string-tree.h>
 #include <workerd/util/string-buffer.h>
+#include <array>
 
 namespace workerd {
 
 namespace {
 
-bool isWhitespace(const char c) {
+constexpr bool isWhitespace(const char c) noexcept {
   return (c == '\r' || c == '\n' || c == '\t' || c == ' ');
```

Comment:
I wonder if this will benefit from a table too?

Reply:
Technically yes, but the trade-off would be the increase in the binary size. I don't have a compelling reason to add or to not add this as a table.